dojo/tools/factory.py

```diff
@@ -23,20 +19,12 @@ def register_parser(scan_type, parser):
     PARSERS[scan_type] = parser
 
 
-def import_parser_factory(file, test, active, verified, scan_type=None):
-    """Return a parser by the scan type
-    This function exists only for backward compatibility
-    """
-    if scan_type in PARSERS:
-        # create dynamicaly in DB
-        test_type, created = Test_Type.objects.get_or_create(name=scan_type)
-        if created:
-            test_type.save()
-        if not test_type.active:
-            raise ValueError(f"Parser {scan_type} is not active")
+def get_parser(scan_type):
+    """Return a parser by the scan type"""
+    if True:
```

Comment:
Why do you use `if True:` here? The `else` path can never be reached.

Reply:
My idea was to add a settings with a regex to enable/disable parser. I forgot it 😅

Reply:
Fixed with my last commits. I added a new setting to filter/allow parsers instead of a always `True` value like in the previous releases.

Reply:
Test_Types have a flag `active`, wouldn't it be better to use that instead of a new setting? Admins can set that easily via the UI instead of editing settings.

Reply:
Made a modification to support the flag `active`. Also I created few unit tests to check that.
You're right, maybe it's better to support both.
Thanks for all your feedback, this make the PR better :)